src/main/resources/assets/js/application/recruiter/controllers/tdprWeekTableController.js

```diff
@@ -87,5 +87,13 @@ define(['angular', 'application/recruiter/tdprRecruiterModule', 'application/rec
 
         $scope.changeSlotTypeCycleThrough = tdprScheduleService.changeSlotTypeCycleThrough;
         $scope.changeSlotDiscardChanges = tdprScheduleService.changeSlotDiscardChanges;
+
+        $scope.isTableEmpty = function () {
+          var persons =  $filter('jobProfileFilter')($scope.persons, $scope.currentJobProfile);
+          if (persons.length === 0)
+            return false;
```

Comment:
name of the function suggests that it returns true if table is empty, but you return false